deploy/hypershift-cluster-monitoring-config/cluster-monitoring-config.yaml

```diff
@@ -0,0 +1,118 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: hypershift-cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    enableUserWorkload: true
+    prometheusK8s:
+    # If the location of this config changes,
+    # make sure to adjust https://github.com/openshift/ops-sop/blob/master/v4/howto/observability/o11m_mst_killswitch.md
+    # so that killswitch instructions point to the right file!
+      remoteWrite:
```

Comment:
We should also keep the existing remoteWrite configs that used to ship metrics to OSD tenant.

Reply:
Won't the metrics being shipped to RHOBS tenant suffice? Why would we need to ship the same to an OSD tenant?

Reply:
Clarified on Slack